content/TrueCommand/TCGettingStarted/Install/Migrate.md

```diff
@@ -58,4 +65,6 @@ To move an existing TrueCommand 1.1 or earlier configuration to a Docker contain
 3. For the TrueCommand 1.1 `/var/db/ixdb/` database, use `pg_dump ixdb > ixdb.sql` to transform the database into a single *.sql* file.
```

Comment:
Lol, this is so incredibly old, I almost think it can be removed. We also have migration running automatically from 2.1->2.2 for influx to rrd and from the format mentioned here back to sql.

If you wanted, you could go straight from 1.2->2.3 by moving `ixdb.sql` to `/data/tc-conf/db`. Not been tested.

Reply:
New article organization provides instruction for installation, updating, and migrating TC using a Docker container, Legacy instructions included here.